client/src/components/OnboardingParent/OnboardingParent.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import "./OnboardingParent.css";
+import logo from "../../../../assets/images/login_logo.png";
+import CarouselDots from "../CarouselDots/CarouselDots";
+// Class Based React Component
+class OnboardingParent extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "OnboardingParent"
+    };
+  }
+
+  // Runs after Component is loaded in the broswer
+  componentDidMount() {}
+
+  // Runs after a component has been updated
+  componentDidUpdate() {}
+
+  // Runs right before a component is removed from the DOM
+  componentWillUnmount() {}
+
+  render() {
+    return (
+      <div className={this.state.classList}>
```

Comment:
Since the state doesn't seem to be changing in this component, you could probably just use the actual className of `OnboardingParent` here directly. Then, you can make this a functional component.